src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(person => (person.sex === 'm')
+  && (century ? Math.ceil(person.died / 100) === century : true));
+
+  const plusAllAges = (prev, current) => prev + (current.died - current.born);
+  const ageSumm = men.reduce(plusAllAges, 0);
+
+  return ageSumm / men.length;
```

Comment:
you do the same on calculateWomenAverageAge. You can create separate function and you it here and on calculateWomenAverageAge